### PR TITLE
NAS-102258 / 11.3 / Update 'guest' help text for SMB shares

### PR DESCRIPTION
**Pull request by anodos325**
- Recent versions of windows may not allow guest SMB2 sessions by
  default.

### CODE REVIEW

File: gui/sharing/models.py [hunk 1]
```diff
@@ -85,7 +85,10 @@ class CIFS_Share(Model):
         verbose_name=_('Allow Guest Access'),
         help_text=_(
             'If true then no password is required to connect to the share. '
-            'Privileges will be those of the guest account.'
+            'Privileges will be those of the guest account. Guest access may '
```
Comment (a-stjohn) [inline gui/sharing/models.py]: This particular sentence might be worded strangely to a user. Perhaps:

```suggestion
            'Privileges belong to the guest account. Guest access may '
```

Comment (wblock) [reply gui/sharing/models.py]: I think it means that privileges will be the same as the guest account.
"may" is the wrong word to use here, it implies permission.  "might" is clearer.  But if it happens every time with those Windows versions, it should just be "is":
```suggestion
            'Privileges are the same as the guest account. Guest access is '
```

File: gui/sharing/models.py [hunk 2]
```diff
@@ -85,7 +85,10 @@ class CIFS_Share(Model):
...
+            'be disabled by default in Windows 10 and Windows Server starting with '
```
Comment (wblock) [inline gui/sharing/models.py]:
```suggestion
            'disabled by default in Windows 10 version 1709 and Windows Server version 1903. '
```

File: gui/sharing/models.py [hunk 3]
```diff
@@ -85,7 +85,10 @@ class CIFS_Share(Model):
...
+            'versions 1709 and 1903. Additional client-side configuration may be '
```
Comment (wblock) [inline gui/sharing/models.py]:
```suggestion
            'Additional client-side configuration is required to provide '
```

File: gui/sharing/models.py [hunk 4]
```diff
@@ -85,7 +85,10 @@ class CIFS_Share(Model):
...
+            'required to provide guest access to these clients. '
```
Comment (wblock) [inline gui/sharing/models.py]:
```suggestion
            'guest access to these clients. See <url with instructions>.'
```

**Pull request merged**